app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -53,11 +68,46 @@ protected void onCreate(Bundle savedInstanceState) {
 
         View infoCard = findViewById(R.id.bottom_card_scroll_view);
         swipeableInfoCard = BottomSheetBehavior.from(infoCard);
+        notification = new Notification(this,AppDatabase.getInstance(this));
+        notification.checkUpCalendarEvery5Minutes();
         showPOICard();
-
         locationFragment = (LocationFragment) getSupportFragmentManager().findFragmentById(R.id.locationFragment);
     }
 
+    public void popUp(CalendarEvent calendarEvent){
+        AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
+        builder.setCancelable(true);
+        builder.setTitle("Heads up... Your next class is in " +  mViewModel.displayTimeToNextClass(calendarEvent.getStartTime()));
+        builder.setMessage("You have " + calendarEvent.getTitle() + " in " + mViewModel.displayTimeToNextClass(calendarEvent.getStartTime())  + " at " +
+                calendarEvent.getLocation() +" ! Please choose to either get directions to your class or to disregard this message");
```

Comment:
Should rename disregard -> ignore
```suggestion
                calendarEvent.getLocation() +" ! Please choose to either get directions to your class or to ignore this message");
```

Reply:
done!